app/[locale]/quizzes/page.tsx

```diff
@@ -43,7 +43,7 @@ export async function generateMetadata({
   return await getMetadata({
     locale,
     slug: ["quizzes"],
-    title: t("common.quizzes-title"),
+    title: `${t("common.quizzes-title")} | ethereum.org`,
```

Comment:
Fine with this for now, but we should tee up a separate task to apply this for all pages from within the `getMetadata` function itself... out-of-scope for here though